### PR TITLE
Fix logic that extracts binary image addresses

### PR DESCRIPTION
**Pull request by ElektrojungeMS**

### CODE REVIEW

File: AppCenterCrashes/AppCenterCrashes/Internals/Util/MSErrorLogFormatter.m [hunk 1]
```diff
@@ -891,25 +886,17 @@ + (MSPLCrashReportThreadInfo *)findCrashedThreadInReport:
...
-
-    for (MSPLCrashReportRegisterInfo *registerInfo in plCrashReporterThread
-             .registers) {
-      [addresses addObject:@(registerInfo.registerValue)];
+    /*
+     * Make sure we don't include the same address twice because it can lead to issues when symbolicating crashes.
+     * See https://github.com/bitstadium/HockeySDK-iOS/pull/203 for the original PR on HockeySDK-iOS and the issue related to
+     * PLCrashReporter at https://groups.google.com/forum/#!msg/plcrashreporter/i6rYc2f4yBo/G8uBTEnMAgAJ.
+     */
+    if ([addresses containsObject:@(info.imageBaseAddress)]) {
```
Comment (dhei) [inline AppCenterCrashes/AppCenterCrashes/Internals/Util/MSErrorLogFormatter.m]: This if...else can be simplified without a `continue`:
```
if (![addresses containsObject:@(info.imageBaseAddress)]) {
  [addresses addObject:@(info.imageBaseAddress)];
}
```

**Comment by ElektrojungeMS** [comment]
Added don't merge because I'm waiting for confirmation by @lumaxis for one thing.

**Comment by ElektrojungeMS** [comment]
We need to revisit this again, as the binary list is way too long and contains unnecessary binaries now. :(

**Comment by jaelim-ms** [comment]
Closing the PR. We don't need this change.

**Pull request closed without merging**